curvefs/docker/debian10/Dockerfile

```diff
@@ -1,11 +1,15 @@
 FROM opencurvedocker/curve-base:debian10
 ENV TZ=Asia/Shanghai
 RUN mkdir -p /curvefs /etc/curvefs /core /etc/curve
-COPY curvefs /curvefs
+COPY curvefs/client /curvefs
+COPY curvefs/conf /curvefs
```

Comment:
Why use multi `COPY` commands to copy curvefs directory?

Reply:
> Why use multi `COPY` commands to copy curvefs directory?

Most of the time, only one or two modules in client, mds or metaserver may be updated, but the entire /curvefs directory needs to be updated, so the entire directory needs to be updated when pushing or pulling.